database/postgres/dal/base.py

```diff
@@ -0,0 +1,8 @@
+
+
+class BaseDAL:
+    def get_query(self, model):
```

Comment:
MINOR: maybe `get_base_query`/`get_raw_query`/`get_generic_query` or something like that